.github/workflows/ci-tests.yaml

```diff
@@ -29,8 +32,8 @@ jobs:
         uses: actions/cache@6998d139ddd3e68c71e9e398d8e40b71a2f39812 # v3.2.5
         with:
           path: /tmp/buildx-cache
-          key: docker-cache-tests
-          restore-keys: docker-cache-master
+          key: docker-cache-tests-v1.23
+          restore-keys: docker-cache-v1.23
```

Comment:
Use `-${{ env.ENVOY_MINOR_RELEASE }}` here as well instead of hard coding the minor version?